demoapps/passwordlogin/build.gradle

```diff
@@ -21,8 +21,8 @@ android {
         }
     }
     compileOptions {
-        targetCompatibility 1.8
-        sourceCompatibility 1.8
+        targetCompatibility JavaVersion.VERSION_1_8
```

Comment:
Why keep 1_8 here? It may be easier for developers to understand the code if they are used to (so not Java 1.8)

Reply:
++ unless there's a good reason to do otherwise, we should be consistent with the Java version

Reply:
Given that is is was not part the public libraries I didn't see the need to remain consistent.  In my mind the passwordlogin example was the "The latest and greatest practices" example, but there's no other motivation I can think of.

I will plan to migrate it in a follow up PR.